nbs/conv_learner.py

```diff
@@ -62,12 +75,15 @@ def __init__(self, data, models, **kwargs):
         self.crit = F.binary_cross_entropy if data.is_multi else F.nll_loss
         if self.metrics is None:
             self.metrics = [accuracy_multi] if self.data.is_multi else [accuracy]
+        if data.is_reg:
+            self.crit = F.mse_loss
+            self.metrics = None
```

Comment:
Don't set this to None if the user explicitly passed in their own choice of metrics. Easiest way is probably to remove this line, and instead protect the 'if self.metrics is None' line above with ' and not data.is_reg'